src/components/LayoutFooter/index.tsx

```diff
@@ -85,8 +85,13 @@ const LayoutFooter: React.FC<Required<ILayoutModifiable>> = ({ modifiers }) => (
               </Link>
             </li>
             <li className={styles.linkItem}>
-              <Link href="/doc/user-guide/privacy" className={styles.link}>
```

Comment:
let's make it a link to the Privacy section? instead of duplicating the menu item

Reply:
Up to you - how would you see it done and linked?

Reply:
Let's keep `/doc/user-guide/privacy`?

Unhide the section + add index.md so that /doc/user-guide/privacy brings you to that page.